### PR TITLE
D35116_D35117_D36092_D35118 Defect Fixed Related Re run Failed Flows …

### PR DESCRIPTION
**Pull request by prashelke**
D35116_D35117_D36092_D35118 Defect Fixed Related Re run Failed Flows from UI

Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [x] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Help Libray document is updated accordingly for Feature changes.

### CODE REVIEW

File: Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs [hunk 1]
```diff
@@ -468,8 +468,8 @@ private void UpdateReRunFailedButtonIcon(bool isRunStarted = false)
                     }
                     else
                     {
-                        xReRunFailedRunsetBtn.ButtonText = "ReRunFailed";
-                        xReRunFailedRunsetBtn.ToolTip = "Run All Failed Runners";
+                        xReRunFailedRunsetBtn.ButtonText = "ReRunFailedFlows";
```
Comment (Maheshkale447) [inline Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs:471]: Change button text to "Re-Run Failed Flows"

File: Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs [hunk 2]
```diff
@@ -468,8 +468,8 @@ private void UpdateReRunFailedButtonIcon(bool isRunStarted = false)
...
+                        xReRunFailedRunsetBtn.ToolTip = "Run All Failed Business flows";
```
Comment (Maheshkale447) [inline Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs:472]: Change Tooltip to "Re-Run All Failed Business flows"

File: Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs [hunk 3]
```diff
@@ -979,11 +979,19 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Gin
...
 
-            if (gingerExecConfig.Runset.RerunConfigurations != null && gingerExecConfig.Runset.RerunConfigurations.Active)
+            if (gingerExecConfig.Runset.RerunConfigurations != null)
             {
-                runSetConfig.ReRunConfigurations.Active = gingerExecConfig.Runset.RerunConfigurations.Active;
-                runSetConfig.ReRunConfigurations.RerunLevel = (global::Ginger.Run.eReRunLevel)gingerExecConfig.Runset.RerunConfigurations.RerunLevel;
-                runSetConfig.ReRunConfigurations.ReferenceExecutionID = gingerExecConfig.Runset.RerunConfigurations.ReferenceExecutionID;
+                if(gingerExecConfig.Runset.RerunConfigurations.Active)
+                {
+                    runSetConfig.ReRunConfigurations.Active = gingerExecConfig.Runset.RerunConfigurations.Active;
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs:986]: Move this line out if as same line exist in else part

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs (hunk 3 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs:986]: done

File: Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs (hunk 2 above)
Comment (prashelke) [reply Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs]: done

File: Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs]: done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**